Dockerfile

```diff
@@ -52,6 +52,33 @@ RUN apt-get update && \
         unzip \
         gcovr
 
+# Add dependencies for other CARMA package
+RUN apt-get install -y \
```

Comment:
I think for ROS packages where possible we'd like to have them tracked in the `package.xml` for CARMABase. Then when we run `rosdep install...` during the CARMABase images install process, it finds them that way. I see some of these (like libpcap-dev) aren't rospackages so they're fine to stay here.

Reply:
I do not even realize we have a package.xml here! That makes sense to me. I can those updates on Monday.